doc/08-Modules.md

```diff
@@ -0,0 +1,68 @@
+# Modules
+
+## Installation
+
+A module should be installed in one of the [configured module paths](03-Configuration.md#general-configuration).
+The default path in most installations is `/usr/share/icingaweb2/modules`.
+
+Each directory in there contains the files for a particular module. The directory's name has to be the one
+that is provided by the module's documentation. If there is none provided, it is usually the name of the
+module in all lowercase. Some modules may use a name prefixed with `icingaweb2-module-` (e.g. `icingaweb2-module-map`).
```

Comment:
An example would be good here for this case too.

(e.g. `icingaweb2-module-map` turns into `map`).